addons/account/models/account_invoice.py

```diff
@@ -40,6 +39,18 @@
 
 MAGIC_COLUMNS = ('id', 'create_uid', 'create_date', 'write_uid', 'write_date')
 
+def calc_check_digits(number):
+    """Calculate the extra digits that should be appended to the number to make it a valid number.
+    Source: python-stdnum iso7064.mod_97_10.calc_check_digits
+    """
+    try:
+        number_base10 = ''.join(str(int(x, 36)) for x in number)
+    except Exception:
+        raise UserError(_("Invalid format for %s") % number)
+
+    checksum = int(number_base10) % 97
+    return '%02d' % ((98 - 100 * checksum) % 97)
```

Comment:
Is `return '%02d' % ((98 - 100 * checksum(number)) % 97)` not enought?

Reply:
but `checksum` is also a oneliner of stdnum which itself called another oneliner `_to_base10` ;-)
https://github.com/arthurdejong/python-stdnum/blob/831c66990ab8fb3040e5b7fc74668836a1d4ef64/stdnum/iso7064/mod_97_10.py#L40-L51